web-test-runner.config.js

```diff
@@ -0,0 +1,16 @@
+export default {
+	files: './test/**/*.test.js',
+	nodeResolve: true,
+	testFramework: {
+		config: {
+			ui: 'bdd',
+			timeout: '10000',
+		}
+	},
+	testRunnerHtml: testFramework =>
+		`<html>
+			<body>
+				<script type="module" src="${testFramework}"></script>
+			</body>
+		</html>`
+};
```

Comment:
I don't think most of this file is needed, if at all. We only normally include it because we inject some extra scripts in there.